stdlib/zipfile/_path/glob.pyi

```diff
@@ -0,0 +1,21 @@
+import re
+import sys
+from collections.abc import Iterator
+
+if sys.version_info >= (3, 13):
+    class Translator:
+        seps: str
+
+        def __init__(self, seps: str = ...) -> None: ...
+        def translate(self, pattern: str) -> str: ...
+        def extend(self, pattern: str) -> None: ...
+        def translate_core(self, pattern: str) -> str: ...
+        def replace(self, match: re.Match[str]) -> str: ...
+        def restrict_rglob(self, pattern: str) -> None: ...
+        def star_not_empty(self, pattern: str) -> str: ...
+
+def separate(pattern: str) -> Iterator[re.Match[str]]: ...
+
+if sys.version_info < (3, 13):
+    def translate(pattern: str) -> str: ...
```

Comment:
I don't see these functions (https://github.com/python/cpython/blob/3.13/Lib/zipfile/_path/glob.py), where did they come from?

I'm also not sure we should be adding this module at all, since it seems to be a private implementation detail. Our policy is to add such modules when people ask for them, but I don't think we need to add them pre-emptively.

Reply:
They exist in 3.12: https://github.com/python/cpython/blob/3.12/Lib/zipfile/_path/glob.py

Reply:
Sounds good, in that case I'll close out this PR, can it be revived if there's a demand here.